README.md

```diff
@@ -1 +1,31 @@
-# programming-dojo
\ No newline at end of file
+# 👩‍💻3AP Days - Programming Dojo👨‍💻
+
+## What will we do?
+
+On Friday afternoon we will improve our programming skills in a pair / extrem programming like setup.
+We will work in setup of 4 - 5 people which will be defined Friday morning.
+
+## What do we need from you
+
+- Please `checkout` this repository
+- Read through the rest of the `README`
+
+## What is a programming dojo
+
+- Please read through this [definition](http://codingdojo.org/WhatIsCodingDojo/)
+- Addtional check out this 📺 [Youtube Video](http://www.youtube.com/watch?v=gav9fLVkZQc)
+
+### Kata
+
+Please through these 3 Katas. Your group will solve at least one of these Kata.
+
+1. http://codingdojo.org/kata/GameOfLife/
+2. http://codingdojo.org/kata/GildedRose/
+3. http://codingdojo.org/kata/TexasHoldEm/
+
+## 🚨Important information
+
+- Each group can decided on their own which language they want to use, e.g. Java, Javascript, Typescript
+- We don't recommend to use any libraries what so ever
```

Comment:
I would allow use of testing libraries (e.g. JUnit, Spock)

Reply:
Good point